WordPress/WordPressTest/SiteCreation/SiteDesignTests.swift

```diff
@@ -73,4 +73,42 @@ class SiteDesignTests: XCTestCase {
         let currentTitle = siteDesignPreviewVC.primaryActionButton.currentTitle
         XCTAssertEqual(expectedPrimaryTitle, currentTitle)
     }
+
+    /// Tests that the preview device on the Design view cannot be changed
+    func testSiteDesignPreviewDeviceIsAlwaysMobile() throws {
+
+        // given
+        let siteDesignVC = SiteDesignContentCollectionViewController(createsSite: false) { _ in }
+        let expectedDevice = PreviewDeviceSelectionViewController.PreviewDevice.mobile
+
+        // when
+        XCTAssertEqual(siteDesignVC.selectedPreviewDevice, expectedDevice)
+        siteDesignVC.selectedPreviewDevice = PreviewDeviceSelectionViewController.PreviewDevice.tablet
+
+        // then
+        XCTAssertEqual(siteDesignVC.selectedPreviewDevice, expectedDevice)
+    }
+
+    /// Tests that the preview device on the Preview view can be changed
+    func testSiteDesignPreviewPreviewDeviceCanBeChanged() throws {
+
+        // given
+        let siteDesignPreviewVC = SiteDesignPreviewViewController(
+            siteDesign: remoteDesign,
+            selectedPreviewDevice: PreviewDeviceSelectionViewController.PreviewDevice.mobile,
+            createsSite: true,
+            onDismissWithDeviceSelected: nil,
+            completion: {design in }
+        )
+        let expectedDevice = PreviewDeviceSelectionViewController.PreviewDevice.tablet
+
+        // when
+        siteDesignPreviewVC.loadViewIfNeeded()
+        siteDesignPreviewVC.viewDidLoad()
+        XCTAssertEqual(siteDesignPreviewVC.selectedPreviewDevice, PreviewDeviceSelectionViewController.PreviewDevice.mobile)
+        siteDesignPreviewVC.selectedPreviewDevice = PreviewDeviceSelectionViewController.PreviewDevice.tablet
+
+        // then
+        XCTAssertEqual(siteDesignPreviewVC.selectedPreviewDevice, expectedDevice)
```

Comment:
Is this just essentially asserting that there's not a noop setter on the preview vc?

Reply:
This one is asserting that on the Preview VC we can change the mode. On the main design VC we do the no op. 👍

Reply:
I ended up removing this test altogether after the last batch of changes as it seemed to lose its value.